modular_zubbers/code/game/objects/items/storage/belt.dm

```diff
@@ -0,0 +1,19 @@
+/obj/item/storage/belt/sabre/centcom
+	name = "commander's sabre sheath"
+	desc = "An incredibly ornate sabre sheath, meant to hold a commander's sabre. The cloth it's made of is soft to the touch, whilst the gold is also of incredibly purity while also alloyed with plasteel."
+	icon = 'modular_zubbers/icons/obj/clothing/belts.dmi'
+	icon_state = "cc_sheath"
+	inhand_icon_state = "cc_sheath"
+	worn_icon_state = "cc_sheath"
```

Comment:
This is failing the worn icon test please re-check that your file are working correctly